x-pack/metricbeat/module/oracle/_meta/docs.asciidoc

```diff
@@ -46,6 +46,18 @@ If the listener is not running, use the command to start:
 
 Then, Metricbeat can be launched.
 
+*Host Configuration*
+
+The following two types of host configurations are supported:
+
+1. Old style host configuration for backwards compatibility: `hosts: ["user/pass@0.0.0.0:1521/ORCLPDB1.localdomain"]`
+
+2. DSN host configuration: `hosts: ['user="user" password="pass" connectString="0.0.0.0:1521/ORCLPDB1.localdomain"']`
```

Comment:
Please include the below mentioned configuration as well

user="user" password="password" connectString="host:port/service_name" sysdba=true

and

"user/password@0.0.0.0:1521/ORCLPDB1.localdomain as sysdba"

These are a bit tricky configs and let us document this to address questions that may arise in futre

Reply:
Sure, updated the documentation. Thanks!